### PR TITLE
Icon background for flows

### PR DESCRIPTION
**Pull request by prazolpp**
## Description

Added component for icons in the tiles, activity list and debug panel for flows.

**Note** this only applies for flows and it doesn't apply for streams

## How has this been tested?
Manually. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build/CI related changes
- [ ] Documentation related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by flogo-oss** [comment]
[CLA assistant check](https://flogo.cla.cloud.tibco.com:443/project-flogo/flogo-web?pullRequest=1319) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://flogo.cla.cloud.tibco.com:443/project-flogo/flogo-web?pullRequest=1319) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://flogo.cla.cloud.tibco.com:443/check/project-flogo/flogo-web?pullRequest=1319) it.</sub>

File: libs/lib-client/activity-icon/src/activity-icon.component.ts [hunk 1]
```diff
@@ -1,4 +1,10 @@
-import { Component, Input, OnChanges, ChangeDetectionStrategy } from '@angular/core';
+import {
```
Comment (prazolpp) [inline libs/lib-client/activity-icon/src/activity-icon.component.ts:1]: This is actually a new file that I copied from another file. You can disregard the older file at the left.

File: libs/lib-client/activity-icon/src/activity-icon.component.less [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+  border-radius: 4px;
+}
+
 .icon {
   margin-right: 8px;
   float: left;
-  width: 24px;
-  height: auto;
+  width: 100%;
+  height: 100%;
```
Comment (fcastill) [inline libs/lib-client/activity-icon/src/activity-icon.component.less:17]: I think this is distorting the image, it should probably be height auto

**Comment by fcastill** [comment]
Alignment/styling issues:

For subflow tile the text looks misaligned:

Screen Shot 2020-08-05 at 3 31 10 PM

For return tile the rounded corners are missing:
Screen Shot 2020-08-05 at 3 32 39 PM

The default icon still looks off from the center, I think it needs to be aligned inside the svg,j ust changing the height won't do
 
Screen Shot 2020-08-05 at 3 32 06 PM

**Pull request merged**